extractor/src/test/resources/damls/EnumMod.daml

```diff
@@ -0,0 +1,33 @@
+-- Copyright (c) 2021 Digital Asset (Switzerland) GmbH and/or its affiliates. All rights reserved.
+-- SPDX-License-Identifier: Apache-2.0
+
+-- Port of daml-lf/encoder/src/test/lf/Enum_all_.lf
```

Comment:
The tests were relying on these Daml-LF tests that use scenario. I don't believe there's an easy way to use script from there, so I ported it. Same for another one.